lib/terminus.spec.js

```diff
@@ -136,16 +136,70 @@ describe('Terminus', () => {
     })
 
     it('returns 503 once signal received', (done) => {
-      execFile('node', ['lib/standalone-tests/terminus.onsignal.fail.js'])
+      let responseAssertionsComplete = false
```

Comment:
I mentioned this up top in the original description, but let me expand upon it:
![image](https://user-images.githubusercontent.com/31864739/46553616-d179c900-c892-11e8-9d5b-54a8f1f79083.png)

I believe it is a best-practice to clean up all forked processes, etc. before calling done() on a test. If we don't wait until the forked http server process is finished before calling done(), then the tests that I added (which run immediately afterwards) will not be able to run at all because port 8000 is still being held up by the forked http server process from this previous test.

Reply:
To clarify further: the http server is open for 1000 ms:
https://github.com/godaddy/terminus/blob/defa4aa3791c818d2f76d69a7c794f6f81043e98/lib/standalone-tests/terminus.onsignal.fail.js#L15

But the test is completed after 300 ms:
https://github.com/godaddy/terminus/blob/defa4aa3791c818d2f76d69a7c794f6f81043e98/lib/terminus.spec.js#L138-L150

..meaning the forked http process is around 700ms after the test has completed, blocking port 8000.